backend/sharelinks/models.py

```
@@ -231,6 +231,9 @@ def __str__(self):
 
     class Meta:
         ordering = ['-viewed_at']
+        indexes = [
+            models.Index(fields=['share_link', '-viewed_at'], name='viewsession_link_viewed_idx'),
+        ]
```

Ruff (0.16.3)

[warning] Mutable default value for class attribute

(RUF012)